docs/STORY-BACKLOG.md

```
@@ -0,0 +1,122 @@
+| Total Items | 6 |
+| TODO | 4 |
+| CANCELLED | 1 |
+| IN PROGRESS | 0 |
+| BLOCKED | 0 |
+| DONE | 1 |
+
+---
+
+## 🔴 HIGH Priority
+
+#### [EPIC12-F1] Shardar stories 12.1-12.12 em arquivos individuais
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)